SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialVirtualWorkerTranslator.cpp

```diff
@@ -231,27 +255,33 @@ void SpatialVirtualWorkerTranslator::QueryForWorkerEntities()
 // returned information will be thrown away.
 void SpatialVirtualWorkerTranslator::WorkerEntityQueryDelegate(const Worker_EntityQueryResponseOp& Op)
 {
+	bWorkerEntityQueryInFlight = false;
+
 	check(StaticComponentView.IsValid());
 	if (!StaticComponentView->HasAuthority(SpatialConstants::INITIAL_VIRTUAL_WORKER_TRANSLATOR_ENTITY_ID, SpatialConstants::VIRTUAL_WORKER_TRANSLATION_COMPONENT_ID))
 	{
 		UE_LOG(LogSpatialVirtualWorkerTranslator, Log, TEXT("(%s) Received response to WorkerEntityQuery, but don't have authority over VIRTUAL_WORKER_MANAGER_COMPONENT.  Aborting processing."), *WorkerId);
```

Comment:
Maybe worth upgrading this log type to something more severe, maybe a warning? I think most of the time at the moment this would be an error but _could_ happen if we start doing periodic queries and the runtime could delegate the translation entity to another worker during gameplay? Whether or not that's true, I'd rather increase the severity then later downgrade if it's spamming.

Reply:
Upgraded to a warning.